TYPO3.Neos/Resources/Private/Templates/Media/Asset/Edit.html

```diff
@@ -71,34 +71,42 @@
 		</div>
 		<div class="neos-footer">
 			<f:link.action action="index" class="neos-button">{neos:backend.translate(id: 'media.cancel', source: 'Modules', package: 'TYPO3.Neos')}</f:link.action>
-			<a title="{neos:backend.translate(id: 'media.clickToDelete', source: 'Modules', package: 'TYPO3.Neos')}" data-toggle="modal" href="#asset-{asset -> f:format.identifier()}" class="neos-button neos-button-danger">{neos:backend.translate(id: 'media.delete', source: 'Modules', package: 'TYPO3.Neos')}</a>
+			<f:if condition="{relatedDocumentNodes}">
+        <f:then>
+          <f:link.action action="relatedDocumentNodes" arguments="{asset:asset}" class="neos-button neos-button-warning">{neos:backend.translate(id: 'media.relatedDocumentNodes', quantity: '{relatedDocumentNodes -> f:count()}', arguments: {0:'{relatedDocumentNodes -> f:count()}'}, source: 'Modules', package: 'TYPO3.Neos')}</f:link.action>
+        </f:then>
+        <f:else>
+          <a title="{neos:backend.translate(id: 'media.clickToDelete', source: 'Modules', package: 'TYPO3.Neos')}" data-toggle="modal" href="#asset-{asset -> f:format.identifier()}" class="neos-button neos-button-danger">{neos:backend.translate(id: 'media.delete', source: 'Modules', package: 'TYPO3.Neos')}</a>
+        </f:else>
+      </f:if>
```

Comment:
I think this might be confusing to users since they look for a way to delete, which isn't possible and the button is hidden. Instead maybe show a disabled delete button with a hover title with the message "Asset could not be deleted, because there are still Nodes using it". And then move the link to displaying the amount of usages to the detail list?

![image](https://cloud.githubusercontent.com/assets/903567/14132087/2548785c-f69e-11e5-90f0-2d8e7d1c5a90.png)

Or if that's not prominent enough

![image](https://cloud.githubusercontent.com/assets/903567/14132095/51d17b12-f69e-11e5-852b-72170e0b42c4.png)